lib/membrane_rtmp_plugin/rtmp/source/interceptor.ex

```diff
@@ -0,0 +1,241 @@
+defmodule Membrane.RTMP.Interceptor do
+  @moduledoc """
+  #{inspect(__MODULE__)} is responsible for parsing
```

Comment:
I think that the meaning of the module has changed a little bit. It is always parsing messages since it returns the media packets as well. Its initial purpose was to stop intercepting once a certain message has been reached.

Also if you have some better name for it be free to suggest it 😉

Reply:
I think `MessageParser` would be good.